docs/en/advanced_tutorials/basedataset.md

```diff
@@ -1,3 +1,503 @@
+
+### The standard data annotation file
+
+In order to unify the dataset interfaces of different tasks and facilitate multiple tasks training in one model, OpenMMLab formulate the **OpenMMLab 2.0 dataset Format specification**. dataset annotation files should conform to this specification, and the BaseDataset reads and parses data annotation files based on this specification. If the data annotation file provided by the user does not conform to the specified format, the user can choose to convert it to the specified format and use OpenMMLab's algorithm toolbox to conduct algorithm training and testing based on the converted data annotation file.
+
+The OpenMMLab 2.0 dataset format specification states that annotation files must be in the format of `json` or `yaml`, `yml` or `pickle`, `pkl`. The dictionary stored in the annotation file must contain two fields, `metainfo` and `data_list`. The `metainfo` is a dictionary containing meta information about the dataset. The `data_list` is a list in which each element is a dictionary and the dictionary defines a raw data info. Each raw data info contains one or more training/test samples.
+
+Here is an example of a JSON annotation file (where each raw data info contains only one training/test sample):
+
+```json
+
+{
+    'metainfo':
+        {
+            'classes': ('cat', 'dog'),
+            ...
+        },
+    'data_list':
+        [
+            {
+                'img_path': "xxx/xxx_0.jpg",
+                'img_label': 0,
+                ...
+            },
+            {
+                'img_path': "xxx/xxx_1.jpg",
+                'img_label': 1,
+                ...
+            },
+            ...
+        ]
+}
+```
+
+We assume that the data is stored in the following path:
+
+```text
+data
+├── annotations
+│   ├── train.json
+├── train
+│   ├── xxx/xxx_0.jpg
+│   ├── xxx/xxx_1.jpg
+│   ├── ...
+```
+
+### The initialization process of the BaseDataset
+
+The initialization process of the BaseDataset is shown as following:
+
+![image](https://user-images.githubusercontent.com/26813582/164611564-af44e3f2-a50f-4ef1-a6db-eddd840e2f40.png)
+
+1. `load metainfo` : Obtain the meta information of the dataset. The meta information can be obtained from three sources with the priority from high to low:
+
+- The dict of `metainfo` passed by the user in the `__init__()` function. The priority is high, since the user can pass this argument when the BaseDataset is instantiated;
```

Comment:
```suggestion
- The dict of `metainfo` passed by the user in the `__init__()` function. The priority is high, since the user can pass this argument when the `BaseDataset` is instantiated;
```